diff --git a/node_modules/json-server/lib/server/defaults.js b/node_modules/json-server/lib/server/defaults.js
index e29a8e1..dbc9ec3 100644
--- a/node_modules/json-server/lib/server/defaults.js
+++ b/node_modules/json-server/lib/server/defaults.js
@@ -17,8 +17,8 @@ const errorhandler = require('errorhandler');
 const bodyParser = require('./body-parser');
 
 module.exports = function (opts) {
-  const userDir = path.join(process.cwd(), 'public');
-  const defaultDir = path.join(__dirname, '../../public');
+  const userDir = path.join(process.cwd(), 'build');
+  const defaultDir = path.join(__dirname, '../../build');
   const staticDir = fs.existsSync(userDir) ? userDir : defaultDir;
   opts = Object.assign({
     logger: true,
